File: tests/files/main/graph.py
if __name__ == "__main__":
    body = parse(sys.argv[1])
    for graph in body.graphs:
        print graph.name
        for adjacency in graph.adjacencies:
            total = 0
            total += adjacency.vertex
            for neighbor in adjacency.neighbors:
                total += neighbor
            print total

File: tests/files/main/graph.java
public static void main(String[] args)
{
    Body body = parse(args[0]);

    for (Graph graph : body.graphs)
    {
        System.out.println(graph.name);
        for (Adjacency adjacency : graph.adjacencies)
        {
            int total = 0;
            total += adjacency.vertex;
            for (int neighbor : adjacency.neighbors)
            {
                total += neighbor;
            }
            System.out.println(total);
        }
    }
}
